torq.q

```diff
@@ -121,17 +152,19 @@ getconfig:{[path;level]
 
         /-get KDBCONFIG path
         conf:`$(kc:getenv[`KDBCONFIG]),"/",path;
-
+				
         /-if level is non-zero return appconfig and config files
         (),$[level;
           appconf,conf;
           first appconf,conf]}
 
+getdepfile:getconfig["config.csv";1]
+
 getconfigfile:getconfig[;0]
 
 version:"1.0"
 application:""
-getversion:{$[0 = count v:@[{first read0 x};hsym`$getenv[`KDBCONFIG],"/version.txt";version];version;v]}
+getversion:{$[0 = count v:@[{raze string ((" S ";enlist ",")0: x)[`version]};hsym`$getenv[`KDBCONFIG],"/config.csv";version];version;v]}
```

Comment:
Is there an assumption on order here?  So if the TorQ version isn't listed first then there will be problems?